src/logging.rs

```diff
@@ -104,6 +108,48 @@ fn is_valid_field(input: &str) -> bool {
     input.chars().all(is_valid_char)
 }
 
+/// A helper for functions that want to take fields as parameters that have already been validated.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+struct ValidField<'a> {
```

Comment:
Is this lifetime ever going be different from `'static`?
If the answer is no, then I think it could be directly declared as such (and possibly elided in some places).

Reply:
Yes, on line 224, the lifetime of the key in `vars: impl Iterator<Item = (K, V)>` isn't given a specific lifetime, so this is needed to match the lifetime within `journal_send`.

Reply:
Ah bummer, sorry for the noise and thanks for the double-check.